presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/StateCompiler.java

```diff
@@ -126,7 +126,7 @@ private StateCompiler()
 
     public static Set<Class<?>> getSupportedFieldTypes()
     {
-        // byte.class and int.class are needed for TriStateBooleanState and BlockViewState respectively
```

Comment:
You just added `BlockViewState` in prev commit, now you remove it.

Anyway, I think listing all the cases where something is needed is short-lived anyway -- next person needing an `int` will notice that `int` is just supported, and won't add their class to this listing.

Reply:
That was a mistake during rebase... I was intended to squash the fixup to the previous commit but apparently I miscounted one...